unit_tests/test_redis_client_base.py

```diff
@@ -0,0 +1,235 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import json
+import uuid
+from unittest.mock import MagicMock, patch
+
+import pytest
+from redis import StrictRedis
+
+import d3a_api_client
+from d3a_api_client.redis_client_base import RedisClientBase, RedisAPIException
+
+AREA_ID = str(uuid.uuid4())
+TRANSACTION_ID = str(uuid.uuid4())
+DEVICE_ID = str(uuid.uuid4())
+
+
+# pylint: disable=protected-access
+class TestRedisClientBase:
+    """Tests for the redis client base behaviour without connecting to d3a."""
```

Comment:
This is actually not a verb, but rather a noun. Since it's a class docstring, it shouldn't be necessary to use imperative verbs as far as I saw in the [PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).